code/test_planning_top/behavior_agent/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>behavior_agent</name>
+  <version>0.0.0</version>
+  <description>The carla scripted package by us</description>
+  <maintainer email="carla.simulator@gmail.com">We</maintainer>
```

Comment:
If we adopt this file into our project, we should change that

Reply:
done in https://github.com/ll7/paf22/pull/170/commits/48f63365232b9939d859252c7f5d7560637d498f